code/modules/clothing/masks/chemmask.dm

```diff
@@ -287,7 +287,7 @@
 
 /obj/item/clothing/mask/chemmask/proc/pack_check(mob/user) //Shuts off mask if the user is not wearing a chempack.
 	var/mob/living/M = user
-	if (!(M && M.back && istype(M.back,/obj/item/weapon/reagent_containers/chempack)))
+	if (!(istype(M) && M.back && istype(M.back,/obj/item/weapon/reagent_containers/chempack)))
 		mask_shutdown(user)
```

Comment:
The code would still get here if the `istype(M)` fails, and proceed to attempt to call `update_inv_wear_mask()` on a non-mob in `mask_shutdown()`. An `istype()` check should be added to that proc as well.